backend/src/nodes/utils/parse_ncnn_param.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+with open("./ncnn_param_schema_converted.json") as f:
+    param_schema = json.load(f)
+
+print(f"{param_schema['Convolution']=}")
+
+
+def parse_ncnn_param(param_path):
```

Comment:
I won't say much about this function. It seems somewhat unpolished, with a few antipatterns sprinkled in here and there. 

Please thoroughly test and type this code. You seem to make a lot of assumptions about the parsed text, so please add assert statements where appropriate.